package.json

```diff
@@ -35,10 +36,13 @@
     "eslint-plugin-react": "7.16.0",
     "npm-run-all": "4.1.5",
     "prettier": "1.19.1",
-    "prop-types": "15.7.2"
+    "prop-types": "15.7.2",
+    "cpx": "^1.5.0",
+    "serve": "^11.2.0"
   },
   "scripts": {
     "build": "./build.sh",
+    "build:dev-html": "./build-dev-html.sh",
```

Comment:
I know the main build script isn't there yet, but it'd be nice to start putting these in scripts/. Maybe call this one "dev" - something that's simpler to type.

Reply:
Sure. I can move the scripts. I called it `dev-html` because it only builds a browser-based version. I thought `dev` would be confusing, since nothing for thunderbird is actually produced...

Reply:
Hmm, how about just `dev-html` then and skip the `build:` prefix?

Reply:
Sound good.